SPECS/xapi-storage-script.spec

```diff
@@ -17,73 +18,56 @@ BuildRequires: message-switch-devel
 BuildRequires: ocaml-rpc-devel
 BuildRequires: xapi-storage-ocaml-plugin-devel
 BuildRequires: ocaml-xcp-rrd-devel
+BuildRequires: systemd-devel
```

Comment:
Just talked about this with @euanh, and we think that -devel is not needed. But `BuildRequires: systemd` is needed to make the systemd install stuff available.